Apps/W1/External File Storage - Azure File Service Connector/app/src/ExtFileShareAccountWizard.Page.al

```diff
@@ -0,0 +1,155 @@
+                ToolTip = 'Specifies the name of the Azure File Share account.';
+                ShowMandatory = true;
+                NotBlank = true;
+
+                trigger OnValidate()
+                begin
+                    IsNextEnabled := FileShareConnectorImpl.IsAccountValid(Rec);
+                end;
+            }
+
+            field(StorageAccountNameField; Rec."Storage Account Name")
+            {
+                Caption = 'Storage Account Name';
+                ShowMandatory = true;
+
+                trigger OnValidate()
+                begin
+                    IsNextEnabled := FileShareConnectorImpl.IsAccountValid(Rec);
+                end;
+            }
+
+            field("Authorization Type"; Rec."Authorization Type")
+            {
+            }
+
+            field(SecretField; Secret)
+            {
+                Caption = 'Secret';
+                ExtendedDatatype = Masked;
+                ToolTip = 'Specifies the Shared access signature Token or SharedKey.';
+                ShowMandatory = true;
+            }
+
+            field(FileShareNameField; Rec."File Share Name")
+            {
+                Caption = 'File Share Name';
+                ToolTip = 'Specifies the file share to use of the storage account.';
+                ShowMandatory = true;
+
+                trigger OnValidate()
+                begin
+                    IsNextEnabled := FileShareConnectorImpl.IsAccountValid(Rec);
+                end;
+            }
+        }
+    }
+
+    actions
+    {
+        area(processing)
+        {
+            action(Back)
+            {
+                Caption = 'Back';
+                ToolTip = 'Back';
+                Image = Cancel;
+                InFooterBar = true;
+
+                trigger OnAction()
+                begin
+                    CurrPage.Close();
+                end;
+            }
+
+            action(Next)
+            {
+                Caption = 'Next';
+                Image = NextRecord;
+                Enabled = IsNextEnabled;
+                InFooterBar = true;
+                ToolTip = 'Next';
+
+                trigger OnAction()
+                begin
+                    FileShareConnectorImpl.CreateAccount(Rec, Secret, FileShareAccount);
+                    CurrPage.Close();
+                end;
+            }
+        }
+    }
+
+    var
+        FileShareAccount: Record "File Account";
+        MediaResources: Record "Media Resources";
+        FileShareConnectorImpl: Codeunit "Ext. File Share Connector Impl";
+        [NonDebuggable]
+        Secret: Text;
+        IsNextEnabled: Boolean;
+        TopBannerVisible: Boolean;
+
+    trigger OnOpenPage()
+    var
+        AssistedSetupLogoTok: Label 'ASSISTEDSETUP-NOTEXT-400PX.PNG', Locked = true;
+    begin
+        Rec.Init();
+        Rec.Insert();
+
+        if MediaResources.Get(AssistedSetupLogoTok) and (CurrentClientType() = ClientType::Web) then
```

Comment:
@JesperSchulz do we already have to prepare for this upcoming change when the Images are removed from the MediaResources table?
https://github.com/microsoft/BCApps/pull/2391